templates/index.html

```diff
@@ -37,7 +37,7 @@
   <!-- Styles -->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/css/materialize.min.css">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="{{ url_for('static', filename='css/dist/styles.min.css') }}" rel="stylesheet">
```

Comment:
I will be removing grunt altogether and trying to move the home page over to using webpack as well. It will simplify the build process. There's still several changes I need to make to the webpack config, so for now I just switched the references to not use minified files and will update the imports once I take care of updating the build process.